spec/schema.json

```diff
@@ -16,53 +16,346 @@
     }
   ],
   "paths": {
-    "/request": {
+    "/status": {
+      "get": {
+        "summary": "Returns information describing the strike-off status of the company",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "200": {
+            "description": "a Status object has been returned successfully",
+            "schema": {
+              "$ref": "#/definitions/StrikeOffStatus"
+            }
+          },
+          "404": {
+            "description": "not found"
+          }
+        }
+      }
+    },
+    "/": {
       "post": {
-        "summary": "Temporary end-point to prove sprint zero connectivity from the web app",
+        "summary": "Create a new strike-off objection request",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "Strike Off Objections"
+        ],
+        "responses": {
+          "201": {
+            "description": "An objection request has been created.",
+            "schema": {
+              "$ref": "#/definitions/ResourceCreated"
+            }
+          }
+        }
+      }
+    },
+    "/{objectionId}": {
+      "parameters": [
+        {
+          "name": "objectionId",
+          "in": "path",
+          "required": true,
+          "description": "ID of the Objection",
+          "type": "string"
+        }
+      ],
+      "get": {
+        "summary": "Returns the objection request identified by objectionId",
```

Comment:
Not an 'objection request' here, just an 'objection'.

Reply:
updated